pom.xml

```diff
@@ -308,20 +309,6 @@
                     </reporting>
                 </configuration>
             </plugin>
-            <plugin>
```

Comment:
GPG plugin is configured in paren pom in oss-release profile. Child project must specify GPG plugin version only. I already did this in my PR.